tox.ini

```diff
@@ -31,11 +31,11 @@ setenv =
   TZ=UTC
 
 [testenv:black]
-commands = black src/ tests/ docker/app/app/
+commands = black src/ tests/ docker/app/
 deps = black
 
 [testenv:lint]
-commands = flake8 src/ tests/ docker/app/app/
+commands = flake8 src/ tests/ docker/app/
 # Pin flake8 to 3.7.9 to match focal
 deps =
```

Comment:
You can probably unpin flake8 version

Reply:
The charm runs only in focal for now, would that be advisable anyway?

Reply:
That shouldn't make any difference. But if you get too much linting problems when unpinning it, we can just postpone it for now